plugin/core/types.py

```diff
@@ -15,6 +15,21 @@
 
 TCP_CONNECT_TIMEOUT = 5
 
+Command = TypedDict('Command', {
+    'title': str,
+    'command': str,
+    'arguments': Optional[List[Any]],
+})
+
+CodeAction = TypedDict('CodeAction', {
+    'title': str,
+    'kind': Optional[str],
+    'diagnostics': Optional[List[Any]],
+    'isPreferred': Optional[bool],
+    'edit': Optional[dict],
+    'command': Optional[Command],
+})
```

Comment:
The best place for the typed dicts are in plugin/core/protocol.py I think :)

Reply:
Moved